packages/storage/src/service.ts

```diff
@@ -151,7 +151,11 @@ export class StorageService implements _FirebaseService {
     /**
      * @internal
      */
-    readonly _url?: string
+    readonly _url?: string,
+    /**
+     * @internal
+     */
+    readonly _firebaseVersion?: string
```

Comment:
Should it be required?

Reply:
Easier to leave the logic intact in `addVersionHeader_` in `request.ts` that replaces it with "AppManager" (or some other default string if that's obsolete) if it is undefined, and not have to change the argument order in StorageService constructor (it would have to come before _url and all the calls to StorageService() have to be rewritten). Just trying not to change too much code at once.